forge-gui/res/adventure/Shandalar/world/enemies.json

```diff
@@ -10029,7 +10029,7 @@
 		"decks/torturer.dck"
 	],
 	"ai": "",
-	"flying": false,
+	"flying": true,
```

Comment:
why it is flying, seems there's only single animation?

Reply:
It's basically to prevent the sprite getting stuck by clutter objects. The sprite also looks like it's floating so it shoudn't be that  strange ?